operating/on_sky.rst

```diff
@@ -0,0 +1,113 @@
+MagAO-X On-sky AO Setup
+==============================
+
+With the instrument installed on the telescope and the dome open, you can proceed with on-sky operations.
+
+The following assumes that you have provided the telescope operator with a target and that you are sitting at the AOC workstation. All GUIs, camera viewers, and the MagAO-X web interface should be open at this point. The GUI layout for on-sky operations on AOC will look something like this: |image1|
+
+If this is the beginning of the night, start from Step 1. Otherwise, proceed to Step 2.

```

Comment:
you need to discuss alignment or link/refer to the startup alignment procedure.  i.o.w. we should specify that the initial conditions are a fully powered on instrument aligned to the internal source.